include/swift/AST/DiagnosticsSema.def

```diff
@@ -5351,6 +5353,10 @@ ERROR(override_nsobject_hashvalue_error,none,
       "'NSObject.hashValue' is not overridable; "
       "did you mean to override 'NSObject.hash'?", ())
 
+ERROR(override_nsobject_hash_error,none,
+      "`NSObject.hash(into:)` is not overridable; "
```

Comment:
Backticks are not used for code voice in diagnostics; please use single quotes instead (see the other diagnostics).